instrumentation/akka-http-10.0/javaagent/akka-http-10.0-javaagent.gradle

```diff
@@ -51,16 +45,9 @@ dependencies {
   library "com.typesafe.akka:akka-http_2.11:10.0.0"
   library "com.typesafe.akka:akka-stream_2.11:2.4.14"
 
-  // There are some internal API changes in 10.1 that we would like to test separately for
-  version101TestImplementation "com.typesafe.akka:akka-http_2.11:10.1.0"
-  version101TestImplementation "com.typesafe.akka:akka-stream_2.11:2.5.11"
-}
-
-test.dependsOn version101Test
-
-compileVersion101TestGroovy {
-  classpath = classpath.plus(files(compileVersion101TestScala.destinationDir))
-  dependsOn compileVersion101TestScala
+  // 10.0.x has some bugs making it incompatible with our test server.
```

Comment:
I tried latest version of 10.0.x too but it still didn't sure. Not sure what the bug is but hopefully this is OK.